pacehrh/R/pace_cadre_process.R

```diff
@@ -232,15 +251,15 @@ SaveCadreAllocations <- function(suiteResults, filepath = NULL, annual = TRUE) {
                         "Trial_num",
                         "Year",
                         "Month",
-                        "Service_time",
+                        "Coverage_service_time",
                         "WeeksPerYr")
```

Comment:
I question the decision to replace Service_time with Coverage_service_time in the cadre allocation CSV file instead of  keeping Service_time and adding a Coverage column so users have more flexibility in their later analysis.

(FWIW I see this as part of the broader story of designing for people who aren't us. When we design for a narrow use case - in this case, analysis that takes coverage into account - we're focusing on the supply side of the problem and under-stating the supply side. If we output all three of Service_time, Coverage and Coverage_service_time, users could do analysis from either direction.)

Reply:
I think we should return both w/wo Coverage as from `pacehrh::SaveExtendedSuiteResults` we can see both columns so it is confusing that it changes after cadre allocation

Reply:
pace_cadre_process.R changed to include both Service_time and Coverage_service_time. Committed and pushed.